src/Compilers/CSharp/Portable/Symbols/SubstitutedNamedTypeSymbol.cs

```diff
@@ -260,7 +270,17 @@ internal sealed override ImmutableArray<Symbol> GetMembersUnordered()
 
             if (IsTupleType)
```

Comment:
>if (IsTupleType) [](http://example.com/codeflow?start=12&length=16)

Is this ```if``` equivalent to the one in the method above? If so, consider extracting into a shared helper.